userspace/src/usbip/usbip_install.c

```diff
@@ -247,6 +268,12 @@ static int usbip_install_base(struct usbip_install_devinfo_struct *data)
 		INSTALLFLAG_FORCE,
 		FALSE);
 	if (!result_ok) {
+		last_error = GetLastError();
+		err("inf file install failed %u", last_error);
```

Comment:
Is the last_error code needed in error message?

Reply:
usbip-win is not at production level. It would be good to debug? 😄 Preferably, I want a hexdecimal error code.

Reply:
oh my bad :) update error message - now it should be more readable